src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/AccessRightService.java

```diff
@@ -96,6 +96,30 @@ public AccessRightService(LocalGroupRepository localGroupRepository, GlobalConfF
         this.endpointService = endpointService;
     }
 
+    public List<AccessRightHolderDto> getAccessRightHoldersByClient(ClientId clientId)
```

Comment:
Would it make sense to split AccessRightService into AccessRightService and ServiceClientService? It's quite big already.

If so, it probably makes sense to do it in XRDDEV-961 branch after refactoring, though.

Reply:
I did think whether I should have created ServiceClientService already in this ticket (created in XRDDEV-961 -branch) but as access right logic for endpoints was also in this service class I decided to move it also here. But I guess it would sense to move the logic to ServiceClientService as this logic is a bit different.

Reply:
Part of the service methods are moved to ServiceClientService in XRDDEV-961